lib/src/view/strand_color_picker.dart

```diff
@@ -36,7 +40,7 @@ mixin StrandColorPickerState on UiState {
 
 class StrandColorPickerComponent
     extends UiStatefulComponent2<StrandColorPickerProps, StrandColorPickerState> {
-  void handleOnChangeComplete(color, event) {
+  void handleOnChangeComplete(JSColor color, event) {
```

Comment:
If the event is not being used, better to name it `_`. (Also, do you know its type?)

Reply:
Not sure what type it event is. Here is the API: https://casesandberg.github.io/react-color/#api-onChangeComplete